packages/runtime/runtime-utils/src/dataStoreHelpers.ts

```diff
@@ -61,6 +62,13 @@ export function responseToException(response: IResponse, request: IRequest): Err
     return responseErr;
 }
 
+// Takes a set of packages and joins them pkg1/pkg2... etc. Tags the field as a code artifact
```

Comment:
Use doc comment format for methods /** */ so that it shows as part of the pubic method. I think that would also get rid of the "// @public (undocumented)" comment in the .md file.